tests/test_state_estimators.py

```diff
@@ -4,6 +4,7 @@
 
 from prog_models import PrognosticsModel
 from prog_algs.exceptions import ProgAlgTypeError
+from prog_algs.state_estimators import KalmanFilter, UnscentedKalmanFilter
```

Comment:
Design question: Is there a reason we import multiple times throughout the file instead of just once at the top?

Reply:
The reason we do that is so that if there's a bug in UKF or KF that causes it to fail when importing- it will show up as a failure in the UKF or KF test. With the import at the top of the file a bug that causes a failure when importing would crash the tests (because it would happen when you first import the file)